tests/lib/rules/no-jquery-methods.js

```diff
@@ -0,0 +1,100 @@
+const rule = require('../../../lib/rules/no-jquery-methods');
+const MESSAGE = rule.meta.message;
+const BLACKLISTMETHOD = rule.meta.blacklist[0];
+const RuleTester = require('eslint').RuleTester;
+const ruleTester = new RuleTester({
+  parserOptions: {
+    ecmaVersion: 6,
+    sourceType: 'module'
+  }
+});
+
+ruleTester.run('no-jquery-methods', rule, {
+  valid: [
+    {
+      code: `
+        export default Ember.Component({
+          init() {
+            Ember.$().notBlacklistedMethod();
+          }
+        });`
+    },
+    {
+      code: `
+        export default Ember.Component({
+          init() {
+            this.$().notBlacklistedMethod();
+          }
+        });`
+    },
+    {
+      code: `
+        export default Ember.Component({
+          init() {
+            const myJQueryObj = this.$();
+            myJQueryObj[${BLACKLISTMETHOD}]();      
```

Comment:
There is a previous discussion about this between @trentmwillis and I. We are intentionally flagging this as `valid`; however I am second guessing this. Essentially why would we ever want to allow a jQuery blacklisted method thru?

I am in agreement (with you) that we should modify this to be invalid. @trentmwillis additional thoughts?